src/main/rx/xbus.c

```diff
@@ -158,9 +181,68 @@ static void xBusUnpackFrame(void)
 
 }
 
+static void xBusUnpackRJ01Frame(void)
+{
+    // Calculate the CRC of the incoming frame
+    uint16_t crc = 0;
+    uint16_t inCrc = 0;
+    uint8_t i = 0;
+    uint16_t value;
+    uint8_t frameAddr;
+
+    // When using the Align RJ01 receiver with 
+    // a MODE B setting in the radio (XG14 tested)
+    // the MODE_B -frame is packed within some
+    // at the moment unknown bytes before and after:
+    // 0xA1 __ __ 0xA1 12*(High + Low) CRC1 CRC2 + __ __ __
```

Comment:
I will look into capture some more for you tomorrow...no time for that now I am afraid.
However, the message I posted is a very accurate sample of what is sent, with a 10ms (if I remember the timing) delay between each new message.

I.e., it looks like this:
BBBBBB...........BBBBBB..........BBBBBB
Where the "BBBBBB" is a frame like the one I posted, the dots represents time with silence.
If I do not move my sticks/switches the message is exactly the same. When I move sticks/switches the "inner message" (i.e. the MODE B frame) is changed as expected.
All..well, I will get some more sample data here for you to dig into. Thanks for that in advance!